stakater-reloader/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# trust-manager container
+
+# Stage1: build from source
+FROM quay.io/cybozu/golang:1.20-jammy AS build
+
+COPY TAG /
+WORKDIR /workspace
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN VERSION=$(cut -d \. -f 1,2,3 < /TAG ) \
+    && curl -fsSL https://github.com/stakater/Reloader/archive/v${VERSION}.tar.gz | \
+    tar --strip-components=1 -xzf -
+
+# Build
+RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=mod -a -o manager main.go
+
+# Stage2: setup runtime container
+FROM quay.io/cybozu/ubuntu:22.04
```

Comment:
How about using scratch for a base image like [trust-manager](https://github.com/cybozu/neco-containers/pull/1044/files#diff-92f8680d6c4eca55cab8c8d0b4521476d79cd3a44cd7916b54af84ab57b82c04R16)?

Reply:
It makes sense since we build the image with `CGO_ENABLED=0`